### DISCUSSION TITLE
Educational discount request rejected

### DISCUSSION THREAD

**Discussion by RohithReddy20** [community/community#40481] [GitHub Education]
![Screenshot_2022-12-01-01-18-27-86_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://user-images.githubusercontent.com/74014434/204894454-9f82db82-820e-438b-96b8-a293404b3d89.jpg)
I'm on semester holidays. How the hell I can be at college. It's just stupid to be rejected based on location.
What should I do now?

**Comment by ossanna16** [comment]
Hi there @RohithReddy20 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
